gdx/src/com/badlogic/gdx/graphics/g2d/Sprite.java

```diff
@@ -529,7 +529,17 @@ public float getX () {
 	public float getY () {
 		return y;
 	}
-
+	
+	/** @return the center x of the sprite, not accounting for scale. */

```

Comment:
I don't want it to account for scale and rotation because the setCenter method does not account for that, also Actors do not account for scale or rotation when you place them via Alignment. I don't want to create another inconsistency here.